ledger/src/blockstore.rs

```diff
@@ -309,7 +321,11 @@ impl Blockstore {
     }
 
     pub fn banking_trace_path(&self) -> PathBuf {
-        self.ledger_path.join("banking_trace")
+        banking_trace_path(&self.ledger_path)
+    }
+
+    pub fn banking_retracer_path(&self) -> PathBuf {
+        banking_retrace_path(&self.ledger_path)
```

Comment:
In general, I don't love these items living in Blockstore. I know it is convenient given that `Blockstore` always has access to `ledger_path`, but these items have nothing to do with Blockstore / rocksdb.

I also get that there isn't a great spot for stuff like that to be shared between ledger-tool / validator / benches. I think the free functions in `blockstore.rs` aren't as bad as methods on `Blockstore`. Maybe a short term compromise is:
- Remove these `Blockstore` methods
- Rename functions like `banking_trace_path` to `banking_trace_path_from_blockstore`, and make that function take a `Blockstore.

I'll mock this up in a separate PR real quick

Reply:
I don't have strong opinion on this.

> I'll mock this up in a separate PR real quick

after this pr landed?

Reply:
Yeah, I did this real quick and wasn't happy with how it looked; will mess around with this after this lands. Resolving this one